styles.scss

```diff
@@ -1,3 +1,6 @@
+---
```

Comment:
These dashes are required otherwise Jekyll doesn't convert it from SCSS to CSS.

It's pretty silly.